### DISCUSSION TITLE
Βαθμολογία στην γραμματεία και στο γκιτχαμπ

### DISCUSSION THREAD

**Discussion by p17balo** [courses-ionio/hci#1498] [Announcements]
Καλησπέρα κύριε καθηγητά, θα μπορούσα να συμπεριληφθώ στην εξεταστική Σεπτεμβρίου για να μπορέσω να στείλω τις εργασίες του μαθήματος στην παρούσα εξεταστική;

**Comment by epidrome** [comment]
κάθε εργασία που στέλνεται σύμφωνα με τις οδηγίες εδώ βαθμολογείται κανονικά από την ομάδα μας

για να περάσουμε βαθμό στο σύστημα της γραμματείας θα πρέπει το όνομα-ΑΜ να είναι διαθέσιμο εκεί, κάτι που δεν εξαρτάται από εμάς, αλλά από τους κανόνες του τμήματος-ιδρύματος

**Comment by p17balo** [comment]
Ευχαριστώ πολυ!
